lib/src/uptech_growthbook_wrapper.dart

```diff
@@ -129,4 +130,8 @@ class UptechGrowthBookWrapper {
       return {};
     }
   }
+
+  String _urlToApiKey({required String url}) {
+    return url.replaceFirst(_fullHostUrl, '');
```

Comment:
I don't believe we want the fallback behavior to be empty string as you can self host GrowthBook in which case the url wouldn't match.

So I think what we need to do is strip off the last part of the url and assume that it is the key and then use the rest of the URL as the `hostURL` in the `GBContext`. This will make it work with people that are self hosting as well as people that are using the hosted solution.